src/pages/index.tsx

```diff
@@ -1,9 +1,8 @@
-import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import React from 'react';
```

Comment:
Why was this file edited? You have changed the functionality of the home page. That is beyond the scope of the issue. Revert.